src/test/unit/ilp.test.js

```diff
@@ -21,7 +21,11 @@ describe('ILP', () => {
     let ilp;
 
     beforeEach(() => {
-        ilp = new Ilp({secret: 'test'});
+        ilp = new Ilp({
+            secret: 'test',
+            // Disable logging for test
+            logger: { log: () => { } }
```

Comment:
logger has also push command, instead of using `empty functions () => {}` let use `jest.fn()` so we can also test what has been logged if needed

Reply:
the logger is mocked multiple times below, so let extract mocking logic to function and reuse it